### PR TITLE
updated for python3 compatibility

### PR DESCRIPTION
**Pull request by Brobin**
Addresses #7 
Fixed print statements
Fixed imports for eg and configparser

### CODE REVIEW

**Comment by srsudar** [comment]
Awesome, thanks! I'll look into merging this in the next day or two.

File: eg/eg_exec.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
+from eg import eg_config
+from eg import eg_util
```
Comment (srsudar) [inline eg/eg_exec.py]: Is this `from eg import ...` relevant to python3, or just a stylistic thing? I want to keep the ability to run `eg_exec.py` as a standalone script, without pip installation, via something like `python eg_exec.py find`, and this `from` syntax seems to cause those invocations to fail.

Comment (Brobin) [inline eg/eg_exec.py]: In Python 3 It was throwign import errorw when installed by pip. I could add a try except for the import errors.

File: eg/eg_exec.py [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 import pydoc
 import sys
 
+try:
+    import eg_config
+    import eg_util
+except ImportError:
+    from eg import eg_config
+    from eg import eg_util
+
```
Comment (srsudar) [inline eg/eg_exec.py]: Maybe we should actually be using [explicit dot syntax](https://www.python.org/dev/peps/pep-0328/#guido-s-decision) for relative imports? I didn't know about this, I've yet to look into it in real depth.

Comment (Brobin) [inline eg/eg_exec.py]: From what I understand, the relative dot imports are really just a stylistic choice. The main advantage is that if you change the package name, you don't have to fix a bunch of imports. If you don't plan on changing the name or wanting to leave that flexibility, there's really no reason to use the dot syntax. I'll leave the decision up to you as the package author.

Comment (srsudar) [inline eg/eg_exec.py]: It looks like the problem stems from the fact that `eg_exec.py` is meant to be run as a script but lives in the `eg` package itself, which [according to Guido](http://stackoverflow.com/questions/16981921/relative-imports-in-python-3) is an antipattern.

Going to merge in the next day or two once I figure out if I agree.

**Pull request merged**